owlsim-services/src/main/java/org/monarchinitiative/owlsim/services/resources/AncestorResource.java

```diff
@@ -39,7 +39,7 @@
   BMKnowledgeBase knowledgeBase;
 
   @GET
-  @Path("/ancestors")
+  @Path("/ancestors/{matcher}")
```

Comment:
Not clear on what this is for; ancestors is a basic query on the kb, no need for a matcher